pandas/core/dtypes/dtypes.py

```diff
@@ -18,7 +20,8 @@
 str_type = str
 
 
-def register_extension_dtype(cls):
+def register_extension_dtype(cls: Type['ExtensionDtype'],
```

Comment:
Same comment on referencing object instead of using a reference

Reply:
agreed, pls change this